README.md

```diff
@@ -70,6 +70,46 @@ Almost everything can be automated through scripts. Process like deploy and dist
 
 Experiences and knowledge must be shared, it's the only way to grow. We love to share what we've learnt so far. We can organize talks or workshops designed for the client needs.
 
+## Design process of a digital product

```

Comment:
In terms of titles structure I would create a `## Design` section and inside this one I'd create the `### Design Process` section. In the future you might want to add more things a part from the process in the Design section.